clients/include/testing_gemv.hpp

```diff
@@ -115,8 +110,20 @@ hipblasStatus_t testing_gemv(Arguments argus)
     /* =====================================================================
            ROCBLAS
     =================================================================== */
-    for(int iter = 0; iter < 1; iter++)
+    hipStream_t stream;
+    status = hipblasGetStream(handle, &stream);
+    if(status != HIPBLAS_STATUS_SUCCESS)
     {
+        hipblasDestroy(handle);
+        return status;
+    }
+    int runs = argus.timing ? argus.cold_iters + argus.iters : 1;
+    for(int iter = 0; iter < runs; iter++)
```

Comment:
I'm wondering if this works correctly when `argus-timing = true && argus.unit_check = true && argus.cold_iters = 2`?
Since we'll be calculating `y = ax + by`, and then using this updated `y` value for `y = ax + by` again.